tests/lax_numpy_test.py

```diff
@@ -1128,6 +1128,23 @@ def testUnique(self, shape, dtype, return_index, return_inverse, return_counts,
     jnp_fun = lambda x: jnp.unique(x, return_index, return_inverse, return_counts)
     self._CheckAgainstNumpy(onp_fun, jnp_fun, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}_{}".format(
+       jtu.format_shape_dtype_string(shape1, dtype1),
+       jtu.format_shape_dtype_string(shape2, dtype2)),
+       "shape1": shape1, "dtype1": dtype1, "shape2": shape2, "dtype2": dtype2,
+       "rng_factory": jtu.rand_default}
+      for dtype1 in [s for s in float_dtypes or int_dtypes if s != jnp.bfloat16]
+      for dtype2 in [s for s in float_dtypes or int_dtypes if s != jnp.bfloat16]
```

Comment:
``float_dtypes or int_dtypes`` is probably not doing what is intended. You can use ``default_dtypes`` to get the union of `float_dtypes` and `int_dtypes`.